### PR TITLE
Add test for loading corrupted metadata

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Add missing setter for promotion policy in test API
Add test for loading corrupted metadata
Update Test Framework

Needs https://github.com/Open-CAS/test-framework/pull/259

### CODE REVIEW

**Comment by ArkadiuszNeumann** [comment]
retest this please

**Review by karolinavelkaja — APPROVED** [review]

**Comment by karolinavelkaja** [comment]
retest this please

File: test/functional/api/cas/cli.py [hunk 1]
```diff
@@ -258,6 +258,23 @@ def set_param_cleaning_acp_cmd(cache_id: str, wake_up: str = None,
...
+
+def set_param_promotion_nhit_cmd(cache_id: str, threshold: str = None,
+                                 trigger: str = None, shortcut: bool = False):
+    add_param = ""
+    if threshold is not None:
+        add_param += (" -t " if shortcut else " --threshold ") + threshold
+    if trigger is not None:
+        add_param += (" -o " if shortcut else " --trigger ") + trigger
+    return _set_param_cmd(namespace="promotion-nhit", cache_id=cache_id,
+                          additional_params=add_param, shortcut=shortcut)
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:275]: added in Kamil's API PR

**Review by Deixx — COMMENTED** [review]
logic should be ok, the corruption could be targeted to specific metadata sectors but even now the test does bring value

**Comment by katlapinka** [comment]
Moved to https://github.com/Open-CAS/open-cas-linux/pull/1524

**Pull request closed without merging**